config.py

```diff
@@ -0,0 +1,17 @@
+import os
+
+
+class Config(object):
+    NAME = os.getenv('NAME', "rm-comms-template")
+    VERSION = os.getenv("VERSION", "0.0.1")
+    SCHEME = os.getenv("SCHEME", "http")
+    HOST = os.getenv("HOST", "0.0.0.0")
+    PORT = os.getenv("PORT", 8081)
+    DEBUG = os.getenv("DEBUG", False)
+    SCHEMA = os.getenv("SCHEMA", "templatesvc")
+    SQLALCHEMY_DATABASE_URI = os.getenv("SQLALCHEMY_DATABASE_URI",
```

Comment:
DATABASE_URI would probably suffice?

Reply:
It's the format required by flask-sqlalchemy

Reply:
Or `DB_URI` if we want to be succinct.

Reply:
Also, putting the version in the config isn't best practice. It should be hardcoded in the root `__init__.py` and updated only on a new release and via a git commit (are we doing release branches @benjefferies @davidmort?).